app/src/main/java/fr/free/nrw/commons/bookmarks/locations/BookmarkLocationsDao.java

```diff
@@ -276,6 +280,15 @@ public static void onUpdate(SQLiteDatabase db, int from, int to) {
                 }
                 return;
             }
+            if (from == 12 && to == 13) {
```

Comment:
this `from == x && to == y` is not how it should be done, typically this is done in a switch on `from` and you just let it fallthrough.

The problem is that we can never guarantee a user upgrades monotonically, a user could theoretically go from v1 to v100 with none of the intervening updates and migrations should happily walk them along that path

Reply:
So should I go ahead and change it to switch with fall through?

Reply:
Yes please